pkg/services/ngalert/notifier/alertmanager_config.go

```diff
@@ -200,6 +186,13 @@ func (moa *MultiOrgAlertmanager) ApplyAlertmanagerConfiguration(ctx context.Cont
 	return nil
 }
 
+type provisioningStore interface {
```

Comment:
Necessary to resolve an import cycle (also removes `github.com/grafana/grafana/pkg/services/ngalert/provisioning` dependency from `ngalert/notifier`)